CRM/Contact/Form/Search/Criteria.php

```diff
@@ -297,6 +297,9 @@ public static function location(&$form) {
 
     $elements = array(
       'street_address' => array(ts('Street Address'), $attributes['street_address'], NULL, NULL),
+      'supplemental_address_1' => array(ts('Supplemental Address 1'), $attributes['supplemental_address_1'], NULL, NULL),
```

Comment:
Hi @eileenmcnaughton thanks for looking at this.  The section you highlighted is a separate change and @yashodha has just merged PR-9850.  That change just added supplemental 1&2 to the search.  This change builds on that to add supplemental 3 to search, alongside all the other supplemental 3 changes